scilpy/tracking/utils.py

```diff
@@ -71,16 +79,13 @@ def add_tracking_options(p):
                          type=float, default=0.1,
                          help='Spherical function relative threshold. '
                               '[%(default)s]')
-    track_g.add_argument('--sh_to_pmf', action='store_true',
-                         help='If set, map sherical harmonics to spherical '
-                              'function (pmf) before tracking (faster, '
-                              'requires more memory)')
     add_sh_basis_args(track_g)
 
     return track_g
 
 
 def add_tracking_ptt_options(p):
+    """ NOT USED IN SCILPY!!! TO DELETE??? """
```

Comment:
Maybe we should take advantage of this PR to either remove or use this function?

Reply:
I asked Gab what to do. He said he would check it. That's why I added it as issue instead.

Reply:
Ok, all good then